src/components/VendorProfileEdit/VendorProfileEdit.jsx

```diff
@@ -0,0 +1,14 @@
+// View for logged-in vendor to edit their details
+// Reached by path '/edit/${vendorUserId}'   --can we use same link for Planner Profile edit but conditionally render based on user type?
+import React from 'react';
+
+
+function VendorProfileEdit() {
```

Comment:
Folks tend to get tripped up on these Edit view -- give me a holler if you want to talk through it.

What I'm going to say: **don't use local state!** Keep track of our changes to the form in Redux. This will allow you to pull the initial values from a saga/from the DB, and not have to duplicate them into local state.

Reply:
I am using reducers no local state required! Going off advice you gave me during the solo project, but I'll let you know!